RevenueCat.xcodeproj/project.pbxproj

```diff
@@ -4515,7 +4515,7 @@
 				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = YES;
 				CODE_SIGN_STYLE = Automatic;
 				INFOPLIST_FILE = Tests/BackendIntegrationTests/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 15.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 16.0;
```

Comment:
We're going to be running these on iOS 17 soon. This simplifies some of the new code, since we never run them on iOS 15 anyway.